jwst/ami/oifits.py

```diff
@@ -579,10 +579,6 @@ def calibrate(self):
         self.calib_oimodel.vis["VISAMP"] = va_out
         self.calib_oimodel.vis["VISAMPERR"] = vaerr_out
 
-        self.calib_oimodel.array["PISTON_T"] = pistons_t
-        self.calib_oimodel.array["PISTON_C"] = pistons_c
-        self.calib_oimodel.array["PIST_ERR"] = pisterr_out
-
         # add calibrated header keywords
```

Comment:
Kneejerk response was "this change doesn't look reasonable", but I see these lines are duplicates of L570:572. I'll wait for regtests, but changes look good.